scheduler/roles/slurm/defaults/main.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+slurm_db_port_default: 3306
+slurm_db_type_default: mariadb
+slurm_db_username_default: root
```

Comment:
we should not take default username. it should be coming from utility

Reply:
![image](https://github.com/user-attachments/assets/1357a8bc-8319-4c0c-9002-aa2a959e15d6)
This is part of slurm_manager vars- retaining the same behaviour